_posts/2021-05-16-overview-of-architecture-review-processes.md

```diff
@@ -0,0 +1,253 @@
+If that happens, they can change the status of the previous ADR to "Superseded".
+
+### Artifacts
+
+The ADR itself is the only artifact in this prototypical architecture review process.
+Reviews on the ADR should be done on the ADR itself when the ADR is created.
+
+In recent years, different templates have appeared for ADR.
+In {% cite Henderson2021 %}, Henderson gathered different ADR example templates.
+However, ADR can be combined with other ways to document architectures such as Design Documents (see also: Design Document or Lightweight Request for Comment Approach):
+"A decision record should not be used to try and capture all of the information relevant to an architecture or design topic. [...] The creators of an architecture or design should author a document that describes it in detail (whether facilitated by a guild or not)." {% cite Kuenzli2019 %}
+
+### Conclusion
+
+ADR are well-suited and widely accepted for documenting architecture decisions {% cite Richards2020 %}.
+The decision log formed by ADR can be a very essential part to understand the history and the current state of an architecture.
+Based on the history and the current state, you can make better decisions in the future.
+ADR and their proposed templates condense architecture decisions to the essential parts.
+Specifically, when they are stored publicly accessible as a basis for discussions, they are a very good tool for architecture work.
+
+ADR and their strict templates are limited when it comes to the bigger picture of changes.
+A good ADR should be written neatly and to the point.
+This, however, may lead to important aspects of changes not being properly described in an ADR.
+"The ADR is a snapshot of information uncovered during the creation process." {% cite Kuenzli2019 %}
+Kuenzli suggests in {% cite Kuenzli2019 %} that also other types of documents should be created in an architecture design process in addition to the ADR.
+For example, Design Documents can complement ADR.
+Design Documents are more verbose and capture more information gathered during research phases.
+The ADR can be created as the outcome of the overall architecture design process.
+
+## Lightweight Request for Comment/Design Document Approach
+
+Another approach is the Lightweight Request for Comment (RFC)/Design Document (DD) approach.
+Essentially, this approach is about writing a DD as the review artifact, sharing this document across the organization in some way, and discussing as well as improving the DD - and the system in design - together with the reviewers.
+It is a more informal way of documenting software architecture at a certain point in time {% cite Ubl2020 %} - originally, considered for the design phase but can be applied to any rework of architectural aspects (solution idea) before the actual (code) work is done.
+
+The Lightweight RFC/DD approach is called differently by various authors - sometimes called Lightweight RFC (or only RFC) and sometimes DD approach.
```

Comment:
```suggestion
The Lightweight RFC/DD approach is called differently by various authors - sometimes Lightweight RFC (or only RFC) and sometimes DD.
```

Reply:
I changed the entire paragraph. Hopefully, to the better ;-)